desk/lib/notes.hoon

```diff
@@ -36,17 +36,14 @@
   (need (get time))
 ::
 ++  reduce
-  |=  [now=time =time del=delta:notes:d]
+  |=  [=time qt=time del=delta:notes:d]
```

Comment:
why can't you just drop the time altogether and have one?

Reply:
there's a difference between something being added vs referencing an existing note/curio/etc. we could handle this higher up to unify I think. basically we need to modify the diff's time to be `now` if it's an add otherwise it's a reference to something existing.

Reply:
ok good

Reply:
I'm doing all this because leaving as sender time felt wrong to me, could end up in bad UX if posts look out of order, etc.